### PR TITLE
Vulture tweaks

### PR DESCRIPTION
**Pull request by Zonespace27**
# About the pull request
- Adds a skillless variant of the vulture
- Lowered spotted/unspotted drift from 33/100 to 25/90
- Lowered scope adjust/move time from 0.5/1.0s to 0.4/0.8s
- Vulture bipod anchors you in place

# Explain why it's good for the game
- Lets admins do memery without wrangling a trait
- Drift is a little *too* overbearing right now, this should make it a bit better.
- Adjusting is hard on a battlefield (as it's meant to be), but it needs to be a little more flexible.
- This is something many bipod users have struggled with, and especially with the vulture sight, considering how much adjustment is done.

# Testing Photographs and Procedure
<details>
<summary>Screenshots & Videos</summary>
tested

</details>

# Changelog
:cl:
admin: Added skillless subtype of the m707
balance: Lowered vulture spotted/unspotted drift from 33/100 to 25/90
balance: Lowered vulture scope adjust/move time from 0.5/1.0s to 0.4/0.8s
balance: Vulture bipod anchors user in place
/:cl:

### CODE REVIEW

File: code/game/objects/structures/vulture_spotter.dm [hunk 1]
```diff
@@ -153,10 +158,13 @@
 		user.pixel_y = 0
 
 /// Handler for when the scope is being attached to the tripod
-/obj/structure/vulture_spotter_tripod/proc/on_scope_attach(mob/user, obj/structure/vulture_spotter_tripod/scope)
+/obj/structure/vulture_spotter_tripod/proc/on_scope_attach(mob/user, obj/item/device/vulture_spotter_scope/scope)
 	if(scope_attached)
 		return
 
+	if(istype(scope, /obj/item/device/vulture_spotter_scope/skillless))
```
Comment (morrowwolf) [inline code/game/objects/structures/vulture_spotter.dm:165]: A var may be faster than a type check but I could be wrong

File: code/modules/cm_marines/equipment/guncases.dm [hunk 2]
```diff
@@ -300,7 +300,7 @@
 	name = "\improper M707 anti-materiel rifle case"
 	desc = "A gun case containing the M707 \"Vulture\" anti-materiel rifle and its requisite spotting tools."
 	icon_state = "guncase_blue"
-	storage_slots = 7
+	storage_slots = 5
```
Comment (morrowwolf) [inline code/modules/cm_marines/equipment/guncases.dm:303]: Should this still be 7 and the skillless at 5?

File: code/modules/projectiles/gun_attachables.dm [hunk 3]
```diff
@@ -1290,7 +1290,7 @@ Defined in conflicts.dm of the #defines folder.
 					return
 				to_chat(scoper, SPAN_NOTICE("You begin adjusting [src]..."))
 				COOLDOWN_START(src, scope_interact_cd, 0.5 SECONDS)
```
Comment (morrowwolf) [inline code/modules/projectiles/gun_attachables.dm:1292]: Should this be changed as well? May be worth making a var

File: code/modules/projectiles/gun_attachables.dm [hunk 4]
```diff
@@ -1308,7 +1308,7 @@ Defined in conflicts.dm of the #defines folder.
 
 				to_chat(scoper, SPAN_NOTICE("You begin moving [src]..."))
 				COOLDOWN_START(src, scope_interact_cd, 1 SECONDS)
```
Comment (morrowwolf) [inline code/modules/projectiles/gun_attachables.dm:1310]: Ditto here

**Review by morrowwolf — CHANGES_REQUESTED** [review]

File: code/modules/cm_marines/equipment/guncases.dm (hunk 2 above)
Comment (Zonespace27) [reply code/modules/cm_marines/equipment/guncases.dm:303]: I really managed to do this backwards
```suggestion
	storage_slots = 7
```

File: code/game/objects/structures/vulture_spotter.dm (hunk 1 above)
Comment (Zonespace27) [reply code/game/objects/structures/vulture_spotter.dm:165]: I'm not too worried about the speed of var vs typecheck here considering this won't happen more than maybe a dozen times in a round

**Pull request merged**